packages/proxy-container/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { createServer } from 'node:http';
+import { proxyRequests } from '@discordjs/proxy';
+import { REST } from '@discordjs/rest';
+
+// We want to let upstream handle retrying
+const api = new REST({ rejectOnRateLimit: () => true, retries: 0 });
```

Comment:
```suggestion
const rest = new REST({ rejectOnRateLimit: () => true, retries: 0 });
```

Reply:
I took the naming from [REST's tests](https://github.com/discordjs/discord.js/blob/3401fd4eb605d093209c6d2d2e2db1f5efce9f52/packages/rest/__tests__/RequestManager.test.ts). This doesn't really matter anyways